2023-07-20 15:51:39,060 INFO    MainThread:23569 [wandb_setup.py:_flush():76] Current SDK version is 0.15.5
2023-07-20 15:51:39,060 INFO    MainThread:23569 [wandb_setup.py:_flush():76] Configure stats pid to 23569
2023-07-20 15:51:39,060 INFO    MainThread:23569 [wandb_setup.py:_flush():76] Loading settings from /home/robot-lab/.config/wandb/settings
2023-07-20 15:51:39,060 INFO    MainThread:23569 [wandb_setup.py:_flush():76] Loading settings from /home/robot-lab/Documents/wandb/settings
2023-07-20 15:51:39,060 INFO    MainThread:23569 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-07-20 15:51:39,060 INFO    MainThread:23569 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-07-20 15:51:39,060 INFO    MainThread:23569 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-07-20 15:51:39,060 INFO    MainThread:23569 [wandb_init.py:_log_setup():507] Logging user logs to /home/robot-lab/Documents/wandb/run-20230720_155139-i9fo6w59/logs/debug.log
2023-07-20 15:51:39,061 INFO    MainThread:23569 [wandb_init.py:_log_setup():508] Logging internal logs to /home/robot-lab/Documents/wandb/run-20230720_155139-i9fo6w59/logs/debug-internal.log
2023-07-20 15:51:39,061 INFO    MainThread:23569 [wandb_init.py:init():547] calling init triggers
2023-07-20 15:51:39,061 INFO    MainThread:23569 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {}
2023-07-20 15:51:39,061 INFO    MainThread:23569 [wandb_init.py:init():571] re-initializing run, found existing run on stack: kec6viui
2023-07-20 15:51:39,062 INFO    MainThread:23569 [wandb_run.py:_finish():1887] finishing run paul-nieuwerburgh/BVAE/kec6viui
2023-07-20 15:51:39,063 INFO    MainThread:23569 [jupyter.py:save_history():445] not saving jupyter history
2023-07-20 15:51:39,063 INFO    MainThread:23569 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-07-20 15:51:39,063 INFO    MainThread:23569 [wandb_init.py:_jupyter_teardown():435] cleaning up jupyter logic
2023-07-20 15:51:39,063 INFO    MainThread:23569 [wandb_run.py:_atexit_cleanup():2121] got exitcode: 0
2023-07-20 15:51:39,063 INFO    MainThread:23569 [wandb_run.py:_restore():2104] restore
2023-07-20 15:51:39,063 INFO    MainThread:23569 [wandb_run.py:_restore():2110] restore done
2023-07-20 15:52:16,997 INFO    MainThread:23569 [wandb_init.py:init():596] starting backend
2023-07-20 15:52:16,997 INFO    MainThread:23569 [wandb_init.py:init():600] setting up manager
2023-07-20 15:52:16,999 INFO    MainThread:23569 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-07-20 15:52:17,000 INFO    MainThread:23569 [wandb_init.py:init():606] backend started and connected
2023-07-20 15:52:17,004 INFO    MainThread:23569 [wandb_run.py:_label_probe_notebook():1233] probe notebook
2023-07-20 15:52:17,004 INFO    MainThread:23569 [wandb_run.py:_label_probe_notebook():1243] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-07-20 15:52:17,004 INFO    MainThread:23569 [wandb_init.py:init():705] updated telemetry
2023-07-20 15:52:17,005 INFO    MainThread:23569 [wandb_init.py:init():738] communicating run to backend with 60.0 second timeout
2023-07-20 15:52:30,484 WARNING MainThread:23569 [wandb_init.py:init():1176] interrupted
Traceback (most recent call last):
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1154, in init
    run = wi.init()
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 741, in init
    result = run_init_handle.wait(
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/threading.py", line 607, in wait
    signaled = self._cond.wait(timeout)
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/threading.py", line 324, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2023-07-20 15:52:37,158 INFO    MainThread:23569 [wandb_run.py:_finish():1887] finishing run paul-nieuwerburgh/BVAE/kec6viui
2023-07-20 15:52:45,747 INFO    MainThread:23569 [wandb_setup.py:_flush():76] Current SDK version is 0.15.5
2023-07-20 15:52:45,748 INFO    MainThread:23569 [wandb_setup.py:_flush():76] Configure stats pid to 23569
2023-07-20 15:52:45,748 INFO    MainThread:23569 [wandb_setup.py:_flush():76] Loading settings from /home/robot-lab/.config/wandb/settings
2023-07-20 15:52:45,748 INFO    MainThread:23569 [wandb_setup.py:_flush():76] Loading settings from /home/robot-lab/Documents/wandb/settings
2023-07-20 15:52:45,748 INFO    MainThread:23569 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-07-20 15:52:45,748 INFO    MainThread:23569 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-07-20 15:52:45,748 INFO    MainThread:23569 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-07-20 15:52:45,748 INFO    MainThread:23569 [wandb_init.py:_log_setup():507] Logging user logs to /home/robot-lab/Documents/wandb/run-20230720_155245-170d9oup/logs/debug.log
2023-07-20 15:52:45,748 INFO    MainThread:23569 [wandb_init.py:_log_setup():508] Logging internal logs to /home/robot-lab/Documents/wandb/run-20230720_155245-170d9oup/logs/debug-internal.log
2023-07-20 15:52:45,748 INFO    MainThread:23569 [wandb_init.py:_jupyter_setup():453] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f86a2447f40>
2023-07-20 15:52:45,748 INFO    MainThread:23569 [wandb_init.py:init():547] calling init triggers
2023-07-20 15:52:45,748 INFO    MainThread:23569 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {}
2023-07-20 15:52:45,748 INFO    MainThread:23569 [wandb_init.py:init():596] starting backend
2023-07-20 15:52:45,748 INFO    MainThread:23569 [wandb_init.py:init():600] setting up manager
2023-07-20 15:52:45,749 INFO    MainThread:23569 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-07-20 15:52:45,749 INFO    MainThread:23569 [wandb_init.py:init():606] backend started and connected
2023-07-20 15:52:45,753 INFO    MainThread:23569 [wandb_run.py:_label_probe_notebook():1233] probe notebook
2023-07-20 15:52:45,753 INFO    MainThread:23569 [wandb_run.py:_label_probe_notebook():1243] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-07-20 15:52:45,753 INFO    MainThread:23569 [wandb_init.py:init():705] updated telemetry
2023-07-20 15:52:45,754 INFO    MainThread:23569 [wandb_init.py:init():738] communicating run to backend with 60.0 second timeout
2023-07-20 15:52:50,460 WARNING MainThread:23569 [wandb_init.py:init():1176] interrupted
Traceback (most recent call last):
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1154, in init
    run = wi.init()
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 741, in init
    result = run_init_handle.wait(
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/threading.py", line 607, in wait
    signaled = self._cond.wait(timeout)
  File "/home/robot-lab/anaconda3/envs/BVAE/lib/python3.10/threading.py", line 324, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
